.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
Perhaps we could use Travis' matrix feature to parallelize the regular test suite with the Crossdock tests.